CHANGELOG.md

```diff
@@ -4,6 +4,63 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginxinc/nginx-gateway-fabric/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 1.5.0
+
+_November 20, 2024_
+
+BREAKING CHANGES:
+
+- NGINX Plus R33 support added. This release of NGINX Plus requires a valid JSON Web Token (JWT) in order to run. Users of NGINX Plus _must_ have this JWT added to a Secret before installing NGINX Gateway Fabric v1.5.0. See the [NGINX Plus JWT](https://docs.nginx.com/nginx-gateway-fabric/installation/nginx-plus-jwt/) guide for information on setting this up. [2760](https://github.com/nginxinc/nginx-gateway-fabric/pull/2760)
+
+FEATURES:
+
+- Add support to retain client IP information. [2284](https://github.com/nginxinc/nginx-gateway-fabric/pull/2284)
+- Add ability to configure data plane log level. [2603](https://github.com/nginxinc/nginx-gateway-fabric/pull/2603)
+- Add support to apply SnippetsFilter in NGINX configuration. [2604](https://github.com/nginxinc/nginx-gateway-fabric/pull/2604)
+- Reduced logging verbosity of default Info log level. [2455](https://github.com/nginxinc/nginx-gateway-fabric/pull/2455)
+
+BUG FIXES:
+
+- Only set stream status zone if hostname exists. [2684](https://github.com/nginxinc/nginx-gateway-fabric/pull/2684)
+- Use correct indentation in lifecycle examples. [2588](https://github.com/nginxinc/nginx-gateway-fabric/pull/2588). Thanks to [Derek F](https://github.com/defrank).
+- Fix an issue with upstream names when split clients are used with a namespace name that starts with a number. [2730](https://github.com/nginxinc/nginx-gateway-fabric/pull/2730)
+- Return 503 when service has no ready endpoints. [2696](https://github.com/nginxinc/nginx-gateway-fabric/pull/2696)
+
+DOCUMENTATION:
+
```

Comment:
I am confused with what change is expected? You think I should just have entries of new doc changes?

Reply:
Sorry for the confusion. I'm suggesting linking to the docs referenced. 

For example:

```
Add [guide](LINK TO GUIDE) for SnippetsFilter.
```

Reply:
This suggestion is based off of the documentation section in our 1.3 release notes: https://github.com/nginxinc/nginx-gateway-fabric/blob/main/CHANGELOG.md#release-130.

Basically, let's add a link to the referenced doc for any net-new document/guide.

Reply:
Yes, makes sense. I updated the links with the doc link for each entry. Expect for the one community PR link, which basically edits the indentation.

Reply:
We still want to link the PR that added the document. My bad for not including that in my example above. 

A couple other things:
- For the "proxyProtocol and rewriteClientIP" doc can we link to the relevant section in the data-plane-configuration.md doc?
- I don't think we need to link to the helm install doc for the "Add description for enabling experimental features" since it's not a large add or net-new doc.

Reply:
I apologize. I didn't see the other comment until later. 
I have fixed it. Linked the document, and the sections along with PR links. 

I have removed entries for non-new documents.